cargo-geiger/Cargo.toml

```diff
@@ -23,6 +23,7 @@ env_logger = "0.7.1"
 geiger = { path = "../geiger", version = "0.4.5" }
 petgraph = "0.5.1"
 pico-args = "0.3.3"
+rand = "0.7.3"
```

Comment:
Can this be a dev-dependency instead?

Reply:
Ahh, apologies, I had meant to remove that, I have taken out in the last commit.

Reply:
Great!